substrate/frame/staking-async/src/session_rotation.rs

```diff
@@ -808,6 +837,11 @@ impl<T: Config> Rotator<T> {
 	fn cleanup_old_era(starting_era: EraIndex) {
 		EraElectionPlanner::<T>::cleanup();
 
+		if let Some(target_era) = starting_era.checked_sub(T::MaxUnbondingDuration::get()) {
+			Eras::<T>::remove_total_unbond_for_era(target_era);
```

Comment:
I don't think we can ever remove this information -- as long as there is anyone that hasn't called withdraw for their old funds, we need to keep them.